src/pages/Admin/Admin_v2.tsx

```diff
@@ -28,13 +26,13 @@ const Admin_v2: React.FC = () => {
   useAdminVenues(user?.uid);
 
   // @debt This selector relies on all venues in firebase being loaded into memory.. not very efficient
-  const venues = useSelector(orderedVenuesSelector);
+  const { relatedVenues, isRelatedVenuesLoading } = useRelatedVenues({});
```

Comment:
I don't think this usage is valid here actually.. I believe the usage here needs to fetch all venues that this user is going to be able to have access to (would have to check code to be certain), not 'related venues'.

Reply:
Replaced with `useAdministeredVenues()` in f97606d53e7ceba1ab4ad0433711a96c16d5b7ea